### ISSUE TITLE
Modify the JsonEditorOptions to prevent type validation?

### ISSUE THREAD

**Issue by bluelbow** [mariohmol/ang-jsoneditor#61]
Is there a way to modify the JsonEditorOptions to prevent type validation? I would like the editor to treat the "value" only as string, is it possible? The editor interprets a string value that is starting by "0" as a number and prevent me from having a leading 0.

Thanks for your help and for the Editor! It's appreciated.

**Comment by mariohmol** [comment]
What you mean? can u give one example?

**Comment by bluelbow** [comment]
Hi, I encountered 2 situations.
1. I have a filed called "TIMESTAMP" and it is a date in a string format. I would be happy if all the field were interpreted as string. But it is validated as a date, but the date doesn't fit (see screenshot)
![date_error](https://user-images.githubusercontent.com/38141276/81302890-f757e880-9048-11ea-8acc-dc21cef2eefd.png).
2. We have a field that looks like this 412340, However, to us, it's as string. So when we try to add a leading 0, it doesn't take it.

In short, I wanted to know if it would be possible for all the field to be interpret as strings.

Thanks.

**Comment by mariohmol** [comment]
I see... I think that's an issue that it's related to the jsoneditor

Maybe this docs can help you, like setting a schema to your editor

https://github.com/josdejong/jsoneditor/blob/master/examples/07_json_schema_validation.html

**Comment by mariohmol** [comment]
This might help you https://github.com/mariohmol/ang-jsoneditor/issues/63#issuecomment-613806162

**Comment by bluelbow** [comment]
Thank you. I will try it.

**Comment by mariohmol** [comment]
cool.. if doesnt work u can reopen
